config.yml

```diff
@@ -1,27 +1,53 @@
 state_abbr: 'IL'
 
 geo_res: 'rto'  # accepts: 'rto' or 'county'
-time_res: 4  # hours 
+time_res: 1  # hours 
+load_filter: 60e3  # MW, load above this level will be removed as outliers.
+total_demand: 136e6  # Annual MWh in the first year
+load_growth: 0.00  # % annual growth
```

Comment:
When choosing a value for this, may be more instructive to consider the total in the final model year and select a rate that grows to that value.

Reply:
Instructive in what way?